src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -2701,6 +2691,223 @@ def test_resource_create_policy_assignment_random(self, resource_group, manageme
+        self.cmd(cmd, checks=[
+            self.check('name', '{psn}'),
+            self.check('displayName', '{psdn}')
+        ])
+
+        # delete the policy set
+        cmd = self.cmdstring('policy set-definition delete -n {psn}', management_group, subscription)
+        self.cmd(cmd)
+        time.sleep(10)  # ensure the policy is gone when run live.
+
+        cmd = self.cmdstring('policy set-definition list', management_group, subscription)
+        self.cmd(cmd, checks=self.check("length([?name=='{psn}'])", 0))
+
+        # delete the policy
+        cmd = self.cmdstring('policy definition delete -n {pn}', management_group, subscription)
+        self.cmd(cmd)
+        time.sleep(10)
+
+        # delete the data policy
+        cmd = self.cmdstring('policy definition delete -n {dpn}', management_group, subscription)
+        self.cmd(cmd)
+        time.sleep(10)
+
+        # ensure the policy is gone when run live.
+        cmd = self.cmdstring('policy definition list', management_group, subscription)
+        self.cmd(cmd, checks=self.check("length([?name=='{pn}'])", 0))
+        self.cmd(cmd, checks=self.check("length([?name=='{dpn}'])", 0))
+
+    @ResourceGroupPreparer(name_prefix='cli_test_policyexemption')
+    @AllowLargeResponse(4096)
+    def test_resource_policyexemption_default(self, resource_group):
+        self.resource_policyexemption_operations(resource_group)
+
+    @ResourceGroupPreparer(name_prefix='cli_test_policyexemption_management_group')
+    @AllowLargeResponse(4096)
+    def test_resource_policyexemption_management_group(self, resource_group):
+        management_group_name = self.create_random_name('cli-test-mgmt-group', 30)
+        self.cmd('account management-group create -n ' + management_group_name)
+        try:
+            self.resource_policyexemption_operations(resource_group, management_group_name)
+        finally:
+            self.cmd('account management-group delete -n ' + management_group_name)
+
+    # mock doesnt work when the subscription comes from --scope, so it cannot be rerecord.
+    @live_only()
+    @ResourceGroupPreparer(name_prefix='cli_test_policyexemption_subscription')
+    @AllowLargeResponse(4096)
+    def test_resource_policyexemption_subscription(self, resource_group):
+        # under playback, we mock it so the subscription id will be '00000000...' and it will match
+        # the same sanitized value in the recording
+        if not self.in_recording:
+            with mock.patch('azure.cli.command_modules.resource.custom._get_subscription_id_from_subscription',
+                            return_value=MOCKED_SUBSCRIPTION_ID):
+                self.resource_policyexemption_operations(resource_group, None, 'f67cc918-f64f-4c3f-aa24-a855465f9d41')
+        else:
+            self.resource_policyexemption_operations(resource_group, None, 'f67cc918-f64f-4c3f-aa24-a855465f9d41')
```

Comment:
Please use the tenant id commonly used by CLI team~
```suggestion
                self.resource_policyexemption_operations(resource_group, None, '0b1f6471-1bf0-4dda-aec3-cb9272f09590')
        else:
            self.resource_policyexemption_operations(resource_group, None, '0b1f6471-1bf0-4dda-aec3-cb9272f09590')
```

Reply:
updated, the mock patch call helped the test pass